seacrowd/sea_datasets/cebuaner/cebuaner.py

```diff
@@ -0,0 +1,151 @@
+from pathlib import Path
+from typing import Dict, Iterable, List, Tuple
+
+import datasets
+from datasets.download.download_manager import DownloadManager
+
+from seacrowd.utils import schemas
+from seacrowd.utils.configs import SEACrowdConfig
+from seacrowd.utils.constants import Licenses, Tasks
+
+_CITATION = """
+@misc{pilar2023cebuaner,
+    title={CebuaNER - A New Baseline Cebuano Named Entity Recognition Model},
+    author={Ma. Beatrice Emanuela Pilar and Ellyza Mari Papas and Mary Loise Buenaventura and Dane Dedoroy and Myron Darrel Montefalcon and Jay Rhald Padilla and Lany Maceda and Mideth Abisado and Joseph Marvin Imperial},
+    year={2023},
+    eprint={2310.00679},
+    archivePrefix={arXiv},
+    primaryClass={cs.CL}
+}
+"""
+
+_LOCAL = False
+_LANGUAGES = ["ceb"]
+_DATASETNAME = "cebuaner"
+_DESCRIPTION = """\
+The CebuaNER dataset contains 4000+ news articles that have been tagged by
+native speakers of Cebuano usin gthe BIO encoding schema for the named entity
+recognition (NER) task.
+"""
+
+_HOMEPAGE = "https://github.com/mebzmoren/CebuaNER"
+_LICENSE = Licenses.CC_BY_NC_SA_4_0.value
+_URL = "https://github.com/mebzmoren/CebuaNER/raw/main/data/annotated_data/final-1.txt"
```

Comment:
Based on my cross-checking to their Github Repo, seems that the file that the authors referring -- have filename ```final.txt``` -- on their [notebook](https://github.com/mebzmoren/CebuaNER/blob/main/notebooks/Named-Entity-Recognition-with-Conditional-Random-Fields.ipynb) is different from what this dataloader impelents -- filename ```final-1.txt``` (and there's also ```final-2.txt``` on their repo under same dir)

open-ended discussion to @holylovenia and @SamuelCahyawijaya 
Is this something that we should clarify to the one who submitted it to SEACrowd or the original author or we can leave them be?

qn to @ljvmiranda921 
do you happen to know their naming convention of their files? (or perhaps sharing the reason on choosing ```final-1.txt``` as the data URL, and not any other files). I'm afraid this may cause issue(s) going forward to their reproducibility and data validity if anyone tries to trace this dataloader deeper to the paper & source code.

Reply:
I contacted the author (@imperialite) and he mentioned that the number corresponds to the identifier of the two annotators who labeled the dataset. The reported reliability score in the paper is quite high so they can be treated as gold-standard.

Reply:
I checked final-1.txt and final-2.txt, they have some differences. I think it is fine if we just load either one here. However, I think we could just make another loader that loads final-2.txt as well. So we have base loader file, one loader that loads final-1.txt, and one loader that loads final-2.txt. What do you think?

Reply:
Sounds good, will do that once I'm back :)

Reply:
Updated. I created separate configs for each annotator. The dataset names now look like this: `cebuaner_annotator_0_source`, `cebuaner_annotator_0_seacrowd_seq_label`, `cebuaner_annotator_1_source`, `cebuaner_annotator_1_seacrowd_seq_label`  https://github.com/SEACrowd/seacrowd-datahub/pull/27/commits/f35b0c9cff3ba6270845152307a3bcdd5d7b4c96

Reply:
In test-suites of SEACrowd, the list of config should contain both `cebuaner_source` and `cebuaner_seacrowd_seq_label`, while the new implementation omits both of them (hence will throw an error under default config of this test cases). Also, this was under discussion w/ CODEOWNERS whether to made it compulsory or not, however from my side it'd be best to re-implement those.
Choosing any of the annotator is okay for me bcs you mentioned that both of them have the high level of alignment, rite?

Reply:
Got it, so to clarify, I need to implement `cebuaner_source` and `cebuaner_seacrowd_seq_label` that loads a single annotator file? In this case I'll just choose annotator 1.

Reply:
Yup!

Reply:
Yup!